stonesoup/simulator/transition.py

```diff
@@ -0,0 +1,266 @@
+# -*- coding: utf-8 -*-
+from datetime import timedelta
+from copy import deepcopy
+
+import numpy as np
+
+from stonesoup.models.transition.linear import ConstantTurn, ConstantVelocity,\
+    CombinedLinearGaussianTransitionModel
+from stonesoup.types.array import StateVector
+
+
+def get_smooth_transition_models(initial_state, x_coords, y_coords, times, turn_rate):
```

Comment:
I wonder if "generate" or "create" would be better than "get" for the function name.